Distinct page for each feature

jseldess opened the pull request:
Fixes #697 

HTML here: http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/df9a869f1fd5db0f829182656b2bc86c238daab0/high-availability.html

@jess-edwards, PTAL

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/docs/699)
<!-- Reviewable:end -->

jess-edwards commented:
@jseldess LGTM. 

Looks like we've got more features than features pages at this point... 

I'll point SQL to the SQL docs, but any ideas about where "highly available by design" should point?

jseldess commented:
@jess-edwards, I added a page for SQL and a page for High Availability, using content from our FAQ. Also updated a few of the other pages with FAQ content so as to not repeat the high-level language on the site homepage. Please take another look.

jess-edwards commented:
@jseldess

You're the best. Thank you!

On Wed, Sep 28, 2016 at 11:29 PM, jseldess <notifications@github.com> wrote:

> @jess-edwards <https://github.com/jess-edwards>, I added a page for SQL
> and a page for High Availability, using content from our FAQ. Also updated
> a few of the other pages with FAQ content so as to not repeat the
> high-level language on the site homepage. Please take another look.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/cockroachdb/docs/pull/699#issuecomment-250361827>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AFwzUolZs710ydZ0pW7m6XuSXWDPuli8ks5quzCwgaJpZM4KJXwF>
> .
>



-- 
Jessica Edwards */ *Director of Marketing & Community
*Cockroach Labs*
